firmware/boron-ins-fsm/src/stateMachine.cpp

```diff
@@ -364,8 +364,8 @@ const char *resetReasonString(int resetReason)
 void getHeartbeat(){
 
     static unsigned long lastHeartbeatPublish = 0;
-    if((millis()-lastHeartbeatPublish) > SM_HEARTBEAT_INTERVAL){
-      
+    if((millis()-lastHeartbeatPublish) > SM_HEARTBEAT_INTERVAL || lastHeartbeatPublish == 0 || (doorHeartbeatFlag && millis()-doorHeartbeatReceived >= 1000)){
```

Comment:
Could you define the `1000` as a constant.

Reply:
Good idea.